### PR TITLE
[ISSUE #4260] Anonymous new ChannelFutureListener() can be replaced with lambda[HelloTask]

### LINKED ISSUE #4260: [Enhancement] Anonymous new ChannelFutureListener() can be replaced with lambda[HelloTask]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/task/HelloTask.java line 99

### Describe the solution you'd like

Replace with lambda.
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by sbmvirdi**
Fixes #4260 .

### Motivation

To contribute in open source

### Modifications

Updated: ChannelFutureListener to lambda expression as its a functional Interface

### Documentation

- Does this pull request introduce a new feature? No
- If yes, how is the feature documented? not applicable
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by mxsm — CHANGES_REQUESTED** [review]
@sbmvirdi Please remove the useless import
image

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
     name: Greeting
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/first-interaction@v1
+      - uses: actions/first-interaction@v1.1.1
         with:
```
Comment (Pil0tXia) [inline .github/workflows/greetings.yml:30]: This modification shouldn't be part of this PR. The good thing is that this update won't affect the current workflow configuration. Acceptable.

**Review by Pil0tXia — APPROVED** [review]

**Review by mxsm — APPROVED** [review]

**Review by xwm1992 — APPROVED** [review]
